app/views/fileChecksResults.scala.html

```diff
@@ -6,6 +6,8 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>
+        @progressIndicator(Messages("fileChecksResults.progress"))
```

Comment:
You've got some progress indicator changes here. Were they mixed up with the beta banner commit by mistake?

Reply:
Because I hadn't merged the update-step-counter branch, I updated the step counter again which wasn't necessary. Thanks for spotting, I'm fixing it!